src/frontend/components/shared/Modal.tsx

```diff
@@ -162,6 +164,7 @@ class Modal extends Component<IModalProps, IModalState> {
     const { show, children, toggle, ROUTE } = this.props
```

Comment:
desctructure out `isThin` here so you don't have to say `this.props.isThin` later